mlos_bench/db/db_schema_pg.sql

```diff
@@ -0,0 +1,108 @@
+--
+-- Copyright (c) Microsoft Corporation.
+-- Licensed under the MIT License.
+--
+-- PostgreSQL version of the DB schema.
+
+DROP TABLE IF EXISTS experiment_config CASCADE;
+DROP TABLE IF EXISTS experiment_merge CASCADE;
+DROP TABLE IF EXISTS trial_status CASCADE;
+DROP TABLE IF EXISTS trial_config CASCADE;
+DROP TABLE IF EXISTS trial_results CASCADE;
+DROP TABLE IF EXISTS trial_telemetry CASCADE;
+
+DROP TYPE IF EXISTS experiment_id_t CASCADE;
+DROP TYPE IF EXISTS trial_id_t CASCADE;
+DROP TYPE IF EXISTS param_id_t CASCADE;
+DROP TYPE IF EXISTS param_value_t CASCADE;
+DROP TYPE IF EXISTS status_t CASCADE;
+
+CREATE DOMAIN experiment_id_t AS VARCHAR(255) NOT NULL;
+CREATE DOMAIN trial_id_t AS INTEGER NOT NULL;
+CREATE DOMAIN param_id_t AS VARCHAR(255) NOT NULL;
+CREATE DOMAIN param_value_t AS VARCHAR(255);
+
+-- Should match the `mlos_bench.environment.Status` enum.
+CREATE TYPE status_t AS ENUM (
+    'UNKNOWN',
+    'PENDING',
+    'READY',
+    'RUNNING',
+    'SUCCEEDED',
+    'CANCELED',
+    'FAILED',
+    'TIMED_OUT'
+);
+
+CREATE TABLE experiment_config (
+    exp_id experiment_id_t,
+    descr TEXT,
+    git_repo VARCHAR(255) NOT NULL,
+    git_commit VARCHAR(40) NOT NULL,
+
+    PRIMARY KEY (exp_id)
+);
+
+CREATE SEQUENCE trial_id_seq;
+
+CREATE TABLE trial_status (
+    exp_id experiment_id_t,
+    trial_id trial_id_t DEFAULT nextval('trial_id_seq'),
+    ts_start TIMESTAMP NOT NULL DEFAULT now(),
+    ts_end TIMESTAMP,
+    status status_t,
+
+    PRIMARY KEY (exp_id, trial_id),
+    FOREIGN KEY (exp_id) REFERENCES experiment_config(exp_id)
+);
+
+ALTER SEQUENCE trial_id_seq owned by trial_status.trial_id;
+
+CREATE TABLE experiment_merge (
+    dest_exp_id experiment_id_t,
+    dest_trial_id trial_id_t,
+    source_exp_id experiment_id_t,
+    source_trial_id trial_id_t,
+
+    UNIQUE (dest_exp_id, dest_trial_id, source_exp_id, source_trial_id),
+
+    FOREIGN KEY (dest_exp_id, dest_trial_id)
+        REFERENCES trial_status(exp_id, trial_id),
+
+    FOREIGN KEY (source_exp_id, source_trial_id)
+        REFERENCES trial_status(exp_id, trial_id)
+);
+
+CREATE TABLE trial_config (
+    exp_id experiment_id_t,
+    trial_id trial_id_t,
+    param_id param_id_t,
+    param_value param_value_t,
+
+    PRIMARY KEY (exp_id, trial_id, param_id),
+    FOREIGN KEY (exp_id, trial_id) REFERENCES trial_status(exp_id, trial_id),
+    FOREIGN KEY (exp_id) REFERENCES experiment_config(exp_id)
+);
+
+CREATE TABLE trial_results (
+    exp_id experiment_id_t,
+    trial_id trial_id_t,
+    param_id param_id_t,
+    param_value param_value_t,
```

Comment:
```suggestion
    score_value score_value_t,
```
?

Reply:
Also, see my comments about the computed scores for FAILED results in #284.
Maybe the right thing to do here is to just submit those scores as `NULL` and let the framework recompute them on demand.

Reply:
yep, I've fixed the other PR to allow bulk loading the status data as well. In the DB schema, however, teh status is associated with the trial and not with the particular value. That is, if some trial fails, simply won't have any records in teh results table